doc/openapi.yaml

```diff
@@ -0,0 +1,104 @@
+openapi: 3.0.0
+info:
+  title: Events API
+  version: 1.0.0
+  description: API pro správu událostí (z minulého úkolu)
+
+servers:
+  - url: http://localhost:4000/api
+
+paths:
+  /events:
+    get:
+      summary: Získat seznam událostí
+      responses:
+        '200':
+          description: Seznam událostí
+          content:
+            application/json:
+              schema:
```

Comment:
pozor, tady ti to neodpovida tvemu serveru, proto ti mozna nesedi typy:
```

app.get("/api/events", (req: Request, res: Response) => {
    res.json({ items: events });
});
```

takze tady nemas ten objekt s `items`. Je potreba pridat jeste jeden typ v teto specce